pkg/gardenlet/controller/shoot/shoot/reconciler_migrate.go

```diff
@@ -172,10 +172,15 @@ func (r *Reconciler) runMigrateShootFlow(ctx context.Context, o *operation.Opera
 			Fn:           flow.TaskFn(botanist.WakeUpKubeAPIServer).DoIf(wakeupRequired),
 			Dependencies: flow.NewTaskIDs(deployETCD, scaleUpETCD, waitUntilControlPlaneReady),
 		})
+		deployGardenerResourceManager = g.Add(flow.Task{
+			Name:         "Deploying gardener-resource-manager (re-run bootstrap if needed)",
+			Fn:           flow.TaskFn(botanist.DeployGardenerResourceManager).DoIf(cleanupShootResources),
+			Dependencies: flow.NewTaskIDs(wakeUpKubeAPIServer),
+		})
 		ensureResourceManagerScaledUp = g.Add(flow.Task{
 			Name:         "Ensuring that the gardener resource manager is scaled to 1",
```

Comment:
Can we also use this PR to change the name of the task here to:
```suggestion
			Name:         "Ensuring that the gardener-resource-manager is scaled to 1",
```
To keep it in line with the name of other similar tasks for GRM.